src/docs/product/integrations/project-mgmt/linear/index.mdx

```diff
@@ -25,3 +25,26 @@ Linear **won't** work with self-hosted Sentry.
    ![](linear-install-pg.png)
 
 2. Follow the full [Linear installation instructions](https://linear.app/docs/sentry).
+
+
+## Issue Management
+
+Issue tracking allows you to create Linear issues from within Sentry, and link Sentry issues to existing Linear issues. Issue management can be configured in two ways - automatically or manually.
+
+#### Automatically
```

Comment:
Always avoid skipping heading levels as this causes the right sidebar to not render properly.

```suggestion
### Automatically
```